packages/hash/frontend/src/components/BlockContextMenu/BlockLoaderInput.tsx

```diff
@@ -24,6 +34,10 @@ export const BlockLoaderInput: React.VFC = () => {
       .then((blockMeta) => {
         setError(null);
         setBlocksMeta((prev) => ({ ...prev, [blockUrl]: blockMeta }));
+        setUserBlocks((prevUserBlocks) => [
+          ...prevUserBlocks,
+          blockMeta.componentMetadata as UserBlock,
+        ]);
```

Comment:
This ought to be wrapped in `unstable_batchedUpdates`

Reply:
I'm assuming this isn't needed once we remove one of the providers here.